// @sitevision/api/types/javax/jcr/query/qom/QueryObjectModelFactory/index.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports["default"] = void 0;
var _default = exports["default"] = {
  createQuery: function createQuery() {},
  selector: function selector() {},
  join: function join() {},
  equiJoinCondition: function equiJoinCondition() {},
  sameNodeJoinCondition: function sameNodeJoinCondition() {},
  childNodeJoinCondition: function childNodeJoinCondition() {},
  descendantNodeJoinCondition: function descendantNodeJoinCondition() {},
  and: function and() {},
  or: function or() {},
  not: function not() {},
  comparison: function comparison() {},
  propertyExistence: function propertyExistence() {},
  fullTextSearch: function fullTextSearch() {},
  sameNode: function sameNode() {},
  childNode: function childNode() {},
  descendantNode: function descendantNode() {},
  propertyValue: function propertyValue() {},
  length: function length() {},
  nodeName: function nodeName() {},
  nodeLocalName: function nodeLocalName() {},
  fullTextSearchScore: function fullTextSearchScore() {},
  lowerCase: function lowerCase() {},
  upperCase: function upperCase() {},
  bindVariable: function bindVariable() {},
  literal: function literal() {},
  ascending: function ascending() {},
  descending: function descending() {},
  column: function column() {}
};
