// @sitevision/api/server/ResourceLocatorUtil/index.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports["default"] = void 0;
var _default = exports["default"] = {
  getRootNode: function getRootNode() {},
  getSite: function getSite() {},
  getSitePage: function getSitePage() {},
  getColorRepository: function getColorRepository() {},
  getFontRepository: function getFontRepository() {},
  getDecorationRepository: function getDecorationRepository() {},
  getFileRepository: function getFileRepository() {},
  getLocalFileRepository: function getLocalFileRepository() {},
  getImageRepository: function getImageRepository() {},
  getLocalImageRepository: function getLocalImageRepository() {},
  getPageCommentRepository: function getPageCommentRepository() {},
  getPersonalFileRepository: function getPersonalFileRepository() {},
  getPersonalImageRepository: function getPersonalImageRepository() {},
  getIconRepository: function getIconRepository() {},
  getIndexRepository: function getIndexRepository() {},
  getDefaultImageRepository: function getDefaultImageRepository() {},
  getDirectoryRepository: function getDirectoryRepository() {},
  getNodeByIdentifier: function getNodeByIdentifier() {},
  getNodeByPath: function getNodeByPath() {},
  getNodeByUrl: function getNodeByUrl() {},
  getNodeId: function getNodeId() {},
  getTemplateRepository: function getTemplateRepository() {},
  getRssFeedRepository: function getRssFeedRepository() {},
  getListStyleRepository: function getListStyleRepository() {},
  getModuleElementDraftRepository: function getModuleElementDraftRepository() {},
  getModuleElementRepository: function getModuleElementRepository() {},
  getModuleElementFileRepository: function getModuleElementFileRepository() {},
  getModuleElementImageRepository: function getModuleElementImageRepository() {},
  getAddonRepository: function getAddonRepository() {},
  getNamedReferenceRepository: function getNamedReferenceRepository() {},
  getRoleRepository: function getRoleRepository() {},
  getVirtualGroupRepository: function getVirtualGroupRepository() {},
  getTagGroupRepository: function getTagGroupRepository() {},
  getResponsiveBreakpointRepository: function getResponsiveBreakpointRepository() {},
  getTopicRepository: function getTopicRepository() {},
  getExternalTopicIntegrationRepository: function getExternalTopicIntegrationRepository() {},
  getOAuth2ConfigurationRepository: function getOAuth2ConfigurationRepository() {},
  getMetadataDefinitionTemplateRepository: function getMetadataDefinitionTemplateRepository() {},
  getTrashcan: function getTrashcan() {},
  getPrincipalRepository: function getPrincipalRepository() {},
  getAliasRepository: function getAliasRepository() {},
  getSiteCookieRepository: function getSiteCookieRepository() {},
  getImageFilterRepository: function getImageFilterRepository() {},
  getWorkStatusTemplateRepository: function getWorkStatusTemplateRepository() {},
  getCssRuleRepository: function getCssRuleRepository() {},
  getLlmConfigurationRepository: function getLlmConfigurationRepository() {},
  getTargetAudienceRepository: function getTargetAudienceRepository() {},
  getSemanticIndexRepository: function getSemanticIndexRepository() {}
};
