// @sitevision/api/types/senselogic/sitevision/api/render/velocity/VelocityAccess.PageUtilConstants/index.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports["default"] = void 0;
var _default = exports["default"] = {
  getPROPERTY_VISIBLE_IN_MENUS: function getPROPERTY_VISIBLE_IN_MENUS() {},
  getPROPERTY_SHORTID: function getPROPERTY_SHORTID() {},
  getPROPERTY_LOCALE: function getPROPERTY_LOCALE() {},
  getPROPERTY_ROBOTS_INDEX: function getPROPERTY_ROBOTS_INDEX() {},
  getPROPERTY_CREATION_DATE: function getPROPERTY_CREATION_DATE() {},
  getPROPERTY_CREATED_BY: function getPROPERTY_CREATED_BY() {},
  getPROPERTY_LAST_MODIFIED_DATE: function getPROPERTY_LAST_MODIFIED_DATE() {},
  getPROPERTY_LAST_MODIFIED_BY: function getPROPERTY_LAST_MODIFIED_BY() {},
  getPROPERTY_LAST_PUBLISHED_BY: function getPROPERTY_LAST_PUBLISHED_BY() {},
  getPROPERTY_LAST_PUBLISH_DATE: function getPROPERTY_LAST_PUBLISH_DATE() {},
  getPROPERTY_PUBLISHED_BY: function getPROPERTY_PUBLISHED_BY() {},
  getPROPERTY_PUBLISH_DATE: function getPROPERTY_PUBLISH_DATE() {},
  getPROPERTY_LAST_UNPUBLISHED_BY: function getPROPERTY_LAST_UNPUBLISHED_BY() {},
  getPROPERTY_LAST_UNPUBLISH_DATE: function getPROPERTY_LAST_UNPUBLISH_DATE() {},
  getPROPERTY_TITLE: function getPROPERTY_TITLE() {}
};
