// @sitevision/api/types/java/nio/file/Path/index.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports["default"] = void 0;
var _default = exports["default"] = {
  getFileSystem: function getFileSystem() {},
  isAbsolute: function isAbsolute() {},
  getRoot: function getRoot() {},
  getFileName: function getFileName() {},
  getParent: function getParent() {},
  getNameCount: function getNameCount() {},
  getName: function getName() {},
  subpath: function subpath() {},
  startsWith: function startsWith() {},
  endsWith: function endsWith() {},
  normalize: function normalize() {},
  resolve: function resolve() {},
  resolveSibling: function resolveSibling() {},
  relativize: function relativize() {},
  toUri: function toUri() {},
  toAbsolutePath: function toAbsolutePath() {},
  toRealPath: function toRealPath() {},
  toFile: function toFile() {},
  register: function register() {},
  iterator: function iterator() {},
  compareTo: function compareTo() {},
  equals: function equals() {},
  hashCode: function hashCode() {},
  toString: function toString() {}
};
