// @sitevision/api/server/PropertyUtil/index.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports["default"] = void 0;
var _default = exports["default"] = {
  getNestedString: function getNestedString() {},
  getNestedStringEscaped: function getNestedStringEscaped() {},
  getNestedInt: function getNestedInt() {},
  getNestedDouble: function getNestedDouble() {},
  getNestedBoolean: function getNestedBoolean() {},
  getNestedNode: function getNestedNode() {},
  getBinary: function getBinary() {},
  getNestedBinary: function getNestedBinary() {},
  getString: function getString() {},
  getStringEscaped: function getStringEscaped() {},
  getInt: function getInt() {},
  getDouble: function getDouble() {},
  getBoolean: function getBoolean() {},
  getNode: function getNode() {},
  getEnabledNode: function getEnabledNode() {},
  getEnabledString: function getEnabledString() {},
  getEnabledStringEscaped: function getEnabledStringEscaped() {},
  getCalendar: function getCalendar() {},
  getNestedCalendar: function getNestedCalendar() {},
  getStrings: function getStrings() {},
  getStringsEscaped: function getStringsEscaped() {},
  getNestedStrings: function getNestedStrings() {},
  getNestedStringsEscaped: function getNestedStringsEscaped() {}
};
