// @sitevision/api/types/java/net/URLConnection/index.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports["default"] = void 0;
var _default = exports["default"] = {
  connect: function connect() {},
  setConnectTimeout: function setConnectTimeout() {},
  getConnectTimeout: function getConnectTimeout() {},
  setReadTimeout: function setReadTimeout() {},
  getReadTimeout: function getReadTimeout() {},
  getURL: function getURL() {},
  getContentLength: function getContentLength() {},
  getContentLengthLong: function getContentLengthLong() {},
  getContentType: function getContentType() {},
  getContentEncoding: function getContentEncoding() {},
  getExpiration: function getExpiration() {},
  getDate: function getDate() {},
  getLastModified: function getLastModified() {},
  getHeaderField: function getHeaderField() {},
  getHeaderFields: function getHeaderFields() {},
  getHeaderFieldInt: function getHeaderFieldInt() {},
  getHeaderFieldLong: function getHeaderFieldLong() {},
  getHeaderFieldDate: function getHeaderFieldDate() {},
  getHeaderFieldKey: function getHeaderFieldKey() {},
  getContent: function getContent() {},
  getPermission: function getPermission() {},
  getInputStream: function getInputStream() {},
  getOutputStream: function getOutputStream() {},
  toString: function toString() {},
  setDoInput: function setDoInput() {},
  getDoInput: function getDoInput() {},
  setDoOutput: function setDoOutput() {},
  getDoOutput: function getDoOutput() {},
  setAllowUserInteraction: function setAllowUserInteraction() {},
  getAllowUserInteraction: function getAllowUserInteraction() {},
  setUseCaches: function setUseCaches() {},
  getUseCaches: function getUseCaches() {},
  setIfModifiedSince: function setIfModifiedSince() {},
  getIfModifiedSince: function getIfModifiedSince() {},
  getDefaultUseCaches: function getDefaultUseCaches() {},
  setDefaultUseCaches: function setDefaultUseCaches() {},
  setRequestProperty: function setRequestProperty() {},
  addRequestProperty: function addRequestProperty() {},
  getRequestProperty: function getRequestProperty() {},
  getRequestProperties: function getRequestProperties() {}
};
