// @sitevision/api/types/senselogic/sitevision/api/collaboration/CollaborationGroupWrapper/index.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports["default"] = void 0;
var _default = exports["default"] = {
  getAdmins: function getAdmins() {},
  isAdmin: function isAdmin() {},
  addAdmin: function addAdmin() {},
  getMembers: function getMembers() {},
  isMember: function isMember() {},
  addMember: function addMember() {},
  removeMember: function removeMember() {},
  getFollowers: function getFollowers() {},
  isFollower: function isFollower() {},
  renameGroup: function renameGroup() {},
  getGroupType: function getGroupType() {},
  setGroupType: function setGroupType() {},
  getGroupState: function getGroupState() {},
  setGroupState: function setGroupState() {},
  getGroupDescription: function getGroupDescription() {},
  setGroupDescription: function setGroupDescription() {},
  getProfileImagesFolder: function getProfileImagesFolder() {},
  getProfileImage: function getProfileImage() {},
  setProfileImage: function setProfileImage() {},
  setLocale: function setLocale() {},
  getCollaborationGroup: function getCollaborationGroup() {}
};
