// @sitevision/api/types/java/time/Instant/index.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports["default"] = void 0;
var _default = exports["default"] = {
  isSupported: function isSupported() {},
  range: function range() {},
  get: function get() {},
  getLong: function getLong() {},
  getEpochSecond: function getEpochSecond() {},
  getNano: function getNano() {},
  "with": function _with() {},
  truncatedTo: function truncatedTo() {},
  plus: function plus() {},
  plusSeconds: function plusSeconds() {},
  plusMillis: function plusMillis() {},
  plusNanos: function plusNanos() {},
  minus: function minus() {},
  minusSeconds: function minusSeconds() {},
  minusMillis: function minusMillis() {},
  minusNanos: function minusNanos() {},
  query: function query() {},
  adjustInto: function adjustInto() {},
  until: function until() {},
  atOffset: function atOffset() {},
  atZone: function atZone() {},
  toEpochMilli: function toEpochMilli() {},
  compareTo: function compareTo() {},
  isAfter: function isAfter() {},
  isBefore: function isBefore() {},
  equals: function equals() {},
  hashCode: function hashCode() {},
  toString: function toString() {}
};
