// @sitevision/api/types/java/util/stream/DoubleStream/index.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports["default"] = void 0;
var _default = exports["default"] = {
  filter: function filter() {},
  map: function map() {},
  mapToObj: function mapToObj() {},
  mapToInt: function mapToInt() {},
  mapToLong: function mapToLong() {},
  flatMap: function flatMap() {},
  distinct: function distinct() {},
  sorted: function sorted() {},
  peek: function peek() {},
  limit: function limit() {},
  skip: function skip() {},
  forEach: function forEach() {},
  forEachOrdered: function forEachOrdered() {},
  toArray: function toArray() {},
  reduce: function reduce() {},
  collect: function collect() {},
  sum: function sum() {},
  min: function min() {},
  max: function max() {},
  count: function count() {},
  average: function average() {},
  summaryStatistics: function summaryStatistics() {},
  anyMatch: function anyMatch() {},
  allMatch: function allMatch() {},
  noneMatch: function noneMatch() {},
  findFirst: function findFirst() {},
  findAny: function findAny() {},
  boxed: function boxed() {},
  sequential: function sequential() {},
  parallel: function parallel() {},
  iterator: function iterator() {},
  spliterator: function spliterator() {},
  builder: function builder() {},
  empty: function empty() {},
  of: function of() {},
  iterate: function iterate() {},
  generate: function generate() {},
  concat: function concat() {}
};
