// @sitevision/api/server/InstanceCreatorUtil/index.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports["default"] = void 0;
var _default = exports["default"] = {
  getDefaultInstance: function getDefaultInstance() {},
  getArgumentInstance: function getArgumentInstance() {},
  getArgumentsInstance: function getArgumentsInstance() {},
  getByteArgument: function getByteArgument() {},
  getShortArgument: function getShortArgument() {},
  getIntArgument: function getIntArgument() {},
  getLongArgument: function getLongArgument() {},
  getFloatArgument: function getFloatArgument() {},
  getDoubleArgument: function getDoubleArgument() {},
  getCharArgument: function getCharArgument() {},
  getBooleanArgument: function getBooleanArgument() {},
  getObjectArgument: function getObjectArgument() {},
  getNullArgument: function getNullArgument() {},
  getCustomArgument: function getCustomArgument() {},
  getEnumArgument: function getEnumArgument() {},
  getEnumInstance: function getEnumInstance() {},
  getList: function getList() {},
  getSet: function getSet() {},
  getMap: function getMap() {},
  getLinkedMap: function getLinkedMap() {},
  getListWrapper: function getListWrapper() {},
  getArraysInstance: function getArraysInstance() {},
  getCollectionsInstance: function getCollectionsInstance() {},
  getMathInstance: function getMathInstance() {},
  getBooleanArray: function getBooleanArray() {},
  getByteArray: function getByteArray() {},
  getCharArray: function getCharArray() {},
  getDoubleArray: function getDoubleArray() {},
  getFloatArray: function getFloatArray() {},
  getIntArray: function getIntArray() {},
  getLongArray: function getLongArray() {},
  getShortArray: function getShortArray() {},
  getStringArray: function getStringArray() {},
  getObjectArray: function getObjectArray() {}
};
