// @sitevision/api/types/java/util/Calendar/index.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports["default"] = void 0;
var _default = exports["default"] = {
  computeTime: function computeTime() {},
  computeFields: function computeFields() {},
  getTime: function getTime() {},
  setTime: function setTime() {},
  getTimeInMillis: function getTimeInMillis() {},
  setTimeInMillis: function setTimeInMillis() {},
  get: function get() {},
  internalGet: function internalGet() {},
  set: function set() {},
  clear: function clear() {},
  isSet: function isSet() {},
  getDisplayName: function getDisplayName() {},
  getDisplayNames: function getDisplayNames() {},
  complete: function complete() {},
  getCalendarType: function getCalendarType() {},
  equals: function equals() {},
  hashCode: function hashCode() {},
  before: function before() {},
  after: function after() {},
  compareTo: function compareTo() {},
  add: function add() {},
  roll: function roll() {},
  setTimeZone: function setTimeZone() {},
  getTimeZone: function getTimeZone() {},
  setLenient: function setLenient() {},
  isLenient: function isLenient() {},
  setFirstDayOfWeek: function setFirstDayOfWeek() {},
  getFirstDayOfWeek: function getFirstDayOfWeek() {},
  setMinimalDaysInFirstWeek: function setMinimalDaysInFirstWeek() {},
  getMinimalDaysInFirstWeek: function getMinimalDaysInFirstWeek() {},
  isWeekDateSupported: function isWeekDateSupported() {},
  getWeekYear: function getWeekYear() {},
  setWeekDate: function setWeekDate() {},
  getWeeksInWeekYear: function getWeeksInWeekYear() {},
  getMinimum: function getMinimum() {},
  getMaximum: function getMaximum() {},
  getGreatestMinimum: function getGreatestMinimum() {},
  getLeastMaximum: function getLeastMaximum() {},
  getActualMinimum: function getActualMinimum() {},
  getActualMaximum: function getActualMaximum() {},
  clone: function clone() {},
  toString: function toString() {},
  toInstant: function toInstant() {}
};
