// @sitevision/api/server/ImageRenderer/index.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports["default"] = void 0;
var _default = exports["default"] = {
  setImageScaler: function setImageScaler() {},
  clearImageScaler: function clearImageScaler() {},
  isImageScalerLoaded: function isImageScalerLoaded() {},
  setUseImageScaler: function setUseImageScaler() {},
  forceUseImageScaler: function forceUseImageScaler() {},
  clearUseImageScaler: function clearUseImageScaler() {},
  setHoverImage: function setHoverImage() {},
  clearHoverImage: function clearHoverImage() {},
  isHoverImageLoaded: function isHoverImageLoaded() {},
  setImage: function setImage() {},
  update: function update() {},
  isImageLoaded: function isImageLoaded() {},
  setUseAutoDescription: function setUseAutoDescription() {},
  forceUseAutoDescription: function forceUseAutoDescription() {},
  clearUseAutoDescription: function clearUseAutoDescription() {},
  setSourceSetMode: function setSourceSetMode() {},
  forceSourceSetMode: function forceSourceSetMode() {},
  clearSourceSetMode: function clearSourceSetMode() {},
  resetSourceSetMode: function resetSourceSetMode() {},
  setDimensionMode: function setDimensionMode() {},
  forceDimensionMode: function forceDimensionMode() {},
  clearDimensionMode: function clearDimensionMode() {},
  resetDimensionMode: function resetDimensionMode() {},
  setUseTitleRendering: function setUseTitleRendering() {},
  forceUseTitleRendering: function forceUseTitleRendering() {},
  clearUseTitleRendering: function clearUseTitleRendering() {},
  setUseAutoTitle: function setUseAutoTitle() {},
  forceUseAutoTitle: function forceUseAutoTitle() {},
  clearUseAutoTitle: function clearUseAutoTitle() {},
  setDescription: function setDescription() {},
  clearDescription: function clearDescription() {},
  setStyle: function setStyle() {},
  clearStyle: function clearStyle() {},
  setUseEncoding: function setUseEncoding() {},
  forceUseEncoding: function forceUseEncoding() {},
  clearUseEncoding: function clearUseEncoding() {},
  setLazyLoad: function setLazyLoad() {},
  forceUseLazyLoad: function forceUseLazyLoad() {},
  clearUseLazyLoad: function clearUseLazyLoad() {},
  addDataAttribute: function addDataAttribute() {},
  clearDataAttributes: function clearDataAttributes() {},
  addAriaAttribute: function addAriaAttribute() {},
  clearAriaAttributes: function clearAriaAttributes() {},
  addCssClass: function addCssClass() {},
  clearCssClasses: function clearCssClasses() {},
  render: function render() {}
};
