// @sitevision/api/types/java/time/LocalTime/index.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports["default"] = void 0;
var _default = exports["default"] = {
  isSupported: function isSupported() {},
  range: function range() {},
  get: function get() {},
  getLong: function getLong() {},
  getHour: function getHour() {},
  getMinute: function getMinute() {},
  getSecond: function getSecond() {},
  getNano: function getNano() {},
  "with": function _with() {},
  withHour: function withHour() {},
  withMinute: function withMinute() {},
  withSecond: function withSecond() {},
  withNano: function withNano() {},
  truncatedTo: function truncatedTo() {},
  plus: function plus() {},
  plusHours: function plusHours() {},
  plusMinutes: function plusMinutes() {},
  plusSeconds: function plusSeconds() {},
  plusNanos: function plusNanos() {},
  minus: function minus() {},
  minusHours: function minusHours() {},
  minusMinutes: function minusMinutes() {},
  minusSeconds: function minusSeconds() {},
  minusNanos: function minusNanos() {},
  query: function query() {},
  adjustInto: function adjustInto() {},
  until: function until() {},
  format: function format() {},
  atDate: function atDate() {},
  atOffset: function atOffset() {},
  toSecondOfDay: function toSecondOfDay() {},
  toNanoOfDay: function toNanoOfDay() {},
  compareTo: function compareTo() {},
  isAfter: function isAfter() {},
  isBefore: function isBefore() {},
  equals: function equals() {},
  hashCode: function hashCode() {},
  toString: function toString() {}
};
