// @sitevision/api/types/java/time/Period/index.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports["default"] = void 0;
var _default = exports["default"] = {
  get: function get() {},
  getUnits: function getUnits() {},
  getChronology: function getChronology() {},
  isZero: function isZero() {},
  isNegative: function isNegative() {},
  getYears: function getYears() {},
  getMonths: function getMonths() {},
  getDays: function getDays() {},
  withYears: function withYears() {},
  withMonths: function withMonths() {},
  withDays: function withDays() {},
  plus: function plus() {},
  plusYears: function plusYears() {},
  plusMonths: function plusMonths() {},
  plusDays: function plusDays() {},
  minus: function minus() {},
  minusYears: function minusYears() {},
  minusMonths: function minusMonths() {},
  minusDays: function minusDays() {},
  multipliedBy: function multipliedBy() {},
  negated: function negated() {},
  normalized: function normalized() {},
  toTotalMonths: function toTotalMonths() {},
  addTo: function addTo() {},
  subtractFrom: function subtractFrom() {},
  equals: function equals() {},
  hashCode: function hashCode() {},
  toString: function toString() {}
};
