// @sitevision/api/types/java/time/chrono/ChronoLocalDate/index.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports["default"] = void 0;
var _default = exports["default"] = {
  timeLineOrder: function timeLineOrder() {},
  from: function from() {},
  getChronology: function getChronology() {},
  getEra: function getEra() {},
  isLeapYear: function isLeapYear() {},
  lengthOfMonth: function lengthOfMonth() {},
  lengthOfYear: function lengthOfYear() {},
  isSupported: function isSupported() {},
  "with": function _with() {},
  plus: function plus() {},
  minus: function minus() {},
  query: function query() {},
  adjustInto: function adjustInto() {},
  until: function until() {},
  format: function format() {},
  atTime: function atTime() {},
  toEpochDay: function toEpochDay() {},
  compareTo: function compareTo() {},
  isAfter: function isAfter() {},
  isBefore: function isBefore() {},
  isEqual: function isEqual() {},
  equals: function equals() {},
  hashCode: function hashCode() {},
  toString: function toString() {}
};
