// @sitevision/api/server/MathInstance/index.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports["default"] = void 0;
var _default = exports["default"] = {
  sin: function sin() {},
  cos: function cos() {},
  tan: function tan() {},
  asin: function asin() {},
  acos: function acos() {},
  atan: function atan() {},
  toRadians: function toRadians() {},
  toDegrees: function toDegrees() {},
  exp: function exp() {},
  log: function log() {},
  log10: function log10() {},
  sqrt: function sqrt() {},
  cbrt: function cbrt() {},
  IEEEremainder: function IEEEremainder() {},
  ceil: function ceil() {},
  floor: function floor() {},
  rint: function rint() {},
  atan2: function atan2() {},
  pow: function pow() {},
  round: function round() {},
  random: function random() {},
  abs: function abs() {},
  max: function max() {},
  min: function min() {},
  ulp: function ulp() {},
  signum: function signum() {},
  sinh: function sinh() {},
  cosh: function cosh() {},
  tanh: function tanh() {},
  hypot: function hypot() {},
  expm1: function expm1() {},
  log1p: function log1p() {},
  copySign: function copySign() {},
  getExponent: function getExponent() {},
  nextAfter: function nextAfter() {},
  nextUp: function nextUp() {},
  scalb: function scalb() {},
  addExact: function addExact() {},
  subtractExact: function subtractExact() {},
  multiplyExact: function multiplyExact() {},
  incrementExact: function incrementExact() {},
  decrementExact: function decrementExact() {},
  negateExact: function negateExact() {},
  toIntExact: function toIntExact() {},
  floorDiv: function floorDiv() {},
  floorMod: function floorMod() {},
  nextDown: function nextDown() {}
};
