// @sitevision/api/server/PermissionUtil/index.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports["default"] = void 0;
var _default = exports["default"] = {
  hasAnonymousReadPermission: function hasAnonymousReadPermission() {},
  hasReadPermission: function hasReadPermission() {},
  hasWritePermission: function hasWritePermission() {},
  hasEffectiveWritePermission: function hasEffectiveWritePermission() {},
  hasPublishPermission: function hasPublishPermission() {},
  hasEffectivePublishPermission: function hasEffectivePublishPermission() {},
  hasWriteSubscriptionPermission: function hasWriteSubscriptionPermission() {},
  hasDeletePermission: function hasDeletePermission() {},
  hasEffectiveDeletePermission: function hasEffectiveDeletePermission() {},
  hasCreatePagePermission: function hasCreatePagePermission() {},
  hasCreateArticlePermission: function hasCreateArticlePermission() {},
  hasCreateArchivePermission: function hasCreateArchivePermission() {},
  hasCreateFolderPermission: function hasCreateFolderPermission() {},
  hasPermission: function hasPermission() {},
  hasPermissions: function hasPermissions() {}
};
