// @sitevision/api/types/java/net/URI/index.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports["default"] = void 0;
var _default = exports["default"] = {
  parseServerAuthority: function parseServerAuthority() {},
  normalize: function normalize() {},
  resolve: function resolve() {},
  relativize: function relativize() {},
  toURL: function toURL() {},
  getScheme: function getScheme() {},
  isAbsolute: function isAbsolute() {},
  isOpaque: function isOpaque() {},
  getRawSchemeSpecificPart: function getRawSchemeSpecificPart() {},
  getSchemeSpecificPart: function getSchemeSpecificPart() {},
  getRawAuthority: function getRawAuthority() {},
  getAuthority: function getAuthority() {},
  getRawUserInfo: function getRawUserInfo() {},
  getUserInfo: function getUserInfo() {},
  getHost: function getHost() {},
  getPort: function getPort() {},
  getRawPath: function getRawPath() {},
  getPath: function getPath() {},
  getRawQuery: function getRawQuery() {},
  getQuery: function getQuery() {},
  getRawFragment: function getRawFragment() {},
  getFragment: function getFragment() {},
  equals: function equals() {},
  hashCode: function hashCode() {},
  compareTo: function compareTo() {},
  toString: function toString() {},
  toASCIIString: function toASCIIString() {}
};
