// @sitevision/api/server/NodeFilterUtil/index.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports["default"] = void 0;
var _default = exports["default"] = {
  getFilteredList: function getFilteredList() {},
  getFilteredValueMap: function getFilteredValueMap() {},
  getFilteredKeyMap: function getFilteredKeyMap() {},
  split: function split() {},
  getCompoundAndFilterBuilder: function getCompoundAndFilterBuilder() {},
  getCompoundOrFilterBuilder: function getCompoundOrFilterBuilder() {},
  getAlwaysAcceptFilter: function getAlwaysAcceptFilter() {},
  getNeverAcceptFilter: function getNeverAcceptFilter() {},
  getNullFilter: function getNullFilter() {},
  getNonNullFilter: function getNonNullFilter() {},
  getSkipFilter: function getSkipFilter() {},
  getInvertedFilter: function getInvertedFilter() {},
  getIdentifierFilter: function getIdentifierFilter() {},
  getIdentifierPrefixFilter: function getIdentifierPrefixFilter() {},
  getIdentifierSuffixFilter: function getIdentifierSuffixFilter() {},
  getPrimaryNodeTypeFilter: function getPrimaryNodeTypeFilter() {},
  getAnyOfPrimaryNodeTypesFilter: function getAnyOfPrimaryNodeTypesFilter() {},
  getNoneOfPrimaryNodeTypesFilter: function getNoneOfPrimaryNodeTypesFilter() {},
  getHasPropertyFilter: function getHasPropertyFilter() {},
  getStringPropertyFilter: function getStringPropertyFilter() {},
  getStringMultiPropertyFilter: function getStringMultiPropertyFilter() {},
  getIgnoreCaseStringPropertyFilter: function getIgnoreCaseStringPropertyFilter() {},
  getStartsWithStringPropertyFilter: function getStartsWithStringPropertyFilter() {},
  getEndsWithStringPropertyFilter: function getEndsWithStringPropertyFilter() {},
  getContainsStringPropertyFilter: function getContainsStringPropertyFilter() {},
  getContainsStringMultiPropertyFilter: function getContainsStringMultiPropertyFilter() {},
  getPatternStringPropertyFilter: function getPatternStringPropertyFilter() {},
  getPatternStringMultiPropertyFilter: function getPatternStringMultiPropertyFilter() {},
  getBooleanPropertyFilter: function getBooleanPropertyFilter() {},
  getNestedBooleanPropertyFilter: function getNestedBooleanPropertyFilter() {},
  getIntPropertyFilter: function getIntPropertyFilter() {},
  getMinIntPropertyFilter: function getMinIntPropertyFilter() {},
  getMaxIntPropertyFilter: function getMaxIntPropertyFilter() {},
  getRangeIntPropertyFilter: function getRangeIntPropertyFilter() {},
  getNestedIntPropertyFilter: function getNestedIntPropertyFilter() {},
  getMinNestedIntPropertyFilter: function getMinNestedIntPropertyFilter() {},
  getMaxNestedIntPropertyFilter: function getMaxNestedIntPropertyFilter() {},
  getRangeNestedIntPropertyFilter: function getRangeNestedIntPropertyFilter() {},
  getDoublePropertyFilter: function getDoublePropertyFilter() {},
  getNestedDoublePropertyFilter: function getNestedDoublePropertyFilter() {},
  getMinDoublePropertyFilter: function getMinDoublePropertyFilter() {},
  getMaxDoublePropertyFilter: function getMaxDoublePropertyFilter() {},
  getRangeDoublePropertyFilter: function getRangeDoublePropertyFilter() {},
  getCalendarPropertyFilter: function getCalendarPropertyFilter() {},
  getNestedCalendarPropertyFilter: function getNestedCalendarPropertyFilter() {},
  getBeforeCalendarPropertyFilter: function getBeforeCalendarPropertyFilter() {},
  getBeforeNestedCalendarPropertyFilter: function getBeforeNestedCalendarPropertyFilter() {},
  getAfterCalendarPropertyFilter: function getAfterCalendarPropertyFilter() {},
  getAfterNestedCalendarPropertyFilter: function getAfterNestedCalendarPropertyFilter() {},
  getRangeCalendarPropertyFilter: function getRangeCalendarPropertyFilter() {},
  getRangeNestedCalendarPropertyFilter: function getRangeNestedCalendarPropertyFilter() {}
};
