// @sitevision/api/types/senselogic/sitevision/api/search/SearchHit/index.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports["default"] = void 0;
var _default = exports["default"] = {
  fieldContains: function fieldContains() {},
  fieldEquals: function fieldEquals() {},
  fieldEqualsIgnoreCase: function fieldEqualsIgnoreCase() {},
  fieldMatches: function fieldMatches() {},
  getNode: function getNode() {},
  getField: function getField() {},
  getFieldEscaped: function getFieldEscaped() {},
  getFields: function getFields() {},
  getFieldsEscaped: function getFieldsEscaped() {},
  getFieldsTruncated: function getFieldsTruncated() {},
  getBooleanField: function getBooleanField() {},
  getBooleanFields: function getBooleanFields() {},
  getDateField: function getDateField() {},
  getDateFields: function getDateFields() {},
  getFloatField: function getFloatField() {},
  getFloatFields: function getFloatFields() {},
  getLongField: function getLongField() {},
  getLongFields: function getLongFields() {},
  getHighlightedField: function getHighlightedField() {},
  getHighlightedFields: function getHighlightedFields() {},
  getFieldTruncated: function getFieldTruncated() {},
  getFieldNames: function getFieldNames() {},
  getScore: function getScore() {},
  isElevated: function isElevated() {},
  getType: function getType() {},
  getClickTrackingCallback: function getClickTrackingCallback() {},
  getClickTrackingUri: function getClickTrackingUri() {}
};
