// @sitevision/api/types/java/time/ZonedDateTime/index.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports["default"] = void 0;
var _default = exports["default"] = {
  isSupported: function isSupported() {},
  range: function range() {},
  get: function get() {},
  getLong: function getLong() {},
  getOffset: function getOffset() {},
  withEarlierOffsetAtOverlap: function withEarlierOffsetAtOverlap() {},
  withLaterOffsetAtOverlap: function withLaterOffsetAtOverlap() {},
  getZone: function getZone() {},
  withZoneSameLocal: function withZoneSameLocal() {},
  withZoneSameInstant: function withZoneSameInstant() {},
  withFixedOffsetZone: function withFixedOffsetZone() {},
  toLocalDateTime: function toLocalDateTime() {},
  toLocalDate: function toLocalDate() {},
  getYear: function getYear() {},
  getMonthValue: function getMonthValue() {},
  getMonth: function getMonth() {},
  getDayOfMonth: function getDayOfMonth() {},
  getDayOfYear: function getDayOfYear() {},
  getDayOfWeek: function getDayOfWeek() {},
  toLocalTime: function toLocalTime() {},
  getHour: function getHour() {},
  getMinute: function getMinute() {},
  getSecond: function getSecond() {},
  getNano: function getNano() {},
  "with": function _with() {},
  withYear: function withYear() {},
  withMonth: function withMonth() {},
  withDayOfMonth: function withDayOfMonth() {},
  withDayOfYear: function withDayOfYear() {},
  withHour: function withHour() {},
  withMinute: function withMinute() {},
  withSecond: function withSecond() {},
  withNano: function withNano() {},
  truncatedTo: function truncatedTo() {},
  plus: function plus() {},
  plusYears: function plusYears() {},
  plusMonths: function plusMonths() {},
  plusWeeks: function plusWeeks() {},
  plusDays: function plusDays() {},
  plusHours: function plusHours() {},
  plusMinutes: function plusMinutes() {},
  plusSeconds: function plusSeconds() {},
  plusNanos: function plusNanos() {},
  minus: function minus() {},
  minusYears: function minusYears() {},
  minusMonths: function minusMonths() {},
  minusWeeks: function minusWeeks() {},
  minusDays: function minusDays() {},
  minusHours: function minusHours() {},
  minusMinutes: function minusMinutes() {},
  minusSeconds: function minusSeconds() {},
  minusNanos: function minusNanos() {},
  query: function query() {},
  until: function until() {},
  format: function format() {},
  toOffsetDateTime: function toOffsetDateTime() {},
  equals: function equals() {},
  hashCode: function hashCode() {},
  toString: function toString() {}
};
