// @sitevision/api/types/java/util/Hashtable/index.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports["default"] = void 0;
var _default = exports["default"] = {
  size: function size() {},
  isEmpty: function isEmpty() {},
  keys: function keys() {},
  elements: function elements() {},
  contains: function contains() {},
  containsValue: function containsValue() {},
  containsKey: function containsKey() {},
  get: function get() {},
  rehash: function rehash() {},
  put: function put() {},
  remove: function remove() {},
  putAll: function putAll() {},
  clear: function clear() {},
  clone: function clone() {},
  toString: function toString() {},
  keySet: function keySet() {},
  entrySet: function entrySet() {},
  values: function values() {},
  equals: function equals() {},
  hashCode: function hashCode() {},
  getOrDefault: function getOrDefault() {},
  forEach: function forEach() {},
  replaceAll: function replaceAll() {},
  putIfAbsent: function putIfAbsent() {},
  replace: function replace() {},
  computeIfAbsent: function computeIfAbsent() {},
  computeIfPresent: function computeIfPresent() {},
  compute: function compute() {},
  merge: function merge() {}
};
