// @sitevision/api/types/java/text/NumberFormat/index.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports["default"] = void 0;
var _default = exports["default"] = {
  format: function format() {},
  parseObject: function parseObject() {},
  parse: function parse() {},
  isParseIntegerOnly: function isParseIntegerOnly() {},
  setParseIntegerOnly: function setParseIntegerOnly() {},
  hashCode: function hashCode() {},
  equals: function equals() {},
  clone: function clone() {},
  isGroupingUsed: function isGroupingUsed() {},
  setGroupingUsed: function setGroupingUsed() {},
  getMaximumIntegerDigits: function getMaximumIntegerDigits() {},
  setMaximumIntegerDigits: function setMaximumIntegerDigits() {},
  getMinimumIntegerDigits: function getMinimumIntegerDigits() {},
  setMinimumIntegerDigits: function setMinimumIntegerDigits() {},
  getMaximumFractionDigits: function getMaximumFractionDigits() {},
  setMaximumFractionDigits: function setMaximumFractionDigits() {},
  getMinimumFractionDigits: function getMinimumFractionDigits() {},
  setMinimumFractionDigits: function setMinimumFractionDigits() {},
  getCurrency: function getCurrency() {},
  setCurrency: function setCurrency() {},
  getRoundingMode: function getRoundingMode() {},
  setRoundingMode: function setRoundingMode() {}
};
