// @sitevision/api/types/java/math/BigDecimal/index.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports["default"] = void 0;
var _default = exports["default"] = {
  add: function add() {},
  subtract: function subtract() {},
  multiply: function multiply() {},
  divide: function divide() {},
  divideToIntegralValue: function divideToIntegralValue() {},
  remainder: function remainder() {},
  divideAndRemainder: function divideAndRemainder() {},
  pow: function pow() {},
  abs: function abs() {},
  negate: function negate() {},
  plus: function plus() {},
  signum: function signum() {},
  scale: function scale() {},
  precision: function precision() {},
  unscaledValue: function unscaledValue() {},
  round: function round() {},
  setScale: function setScale() {},
  movePointLeft: function movePointLeft() {},
  movePointRight: function movePointRight() {},
  scaleByPowerOfTen: function scaleByPowerOfTen() {},
  stripTrailingZeros: function stripTrailingZeros() {},
  compareTo: function compareTo() {},
  equals: function equals() {},
  min: function min() {},
  max: function max() {},
  hashCode: function hashCode() {},
  toString: function toString() {},
  toEngineeringString: function toEngineeringString() {},
  toPlainString: function toPlainString() {},
  toBigInteger: function toBigInteger() {},
  toBigIntegerExact: function toBigIntegerExact() {},
  longValue: function longValue() {},
  longValueExact: function longValueExact() {},
  intValue: function intValue() {},
  intValueExact: function intValueExact() {},
  shortValueExact: function shortValueExact() {},
  byteValueExact: function byteValueExact() {},
  floatValue: function floatValue() {},
  doubleValue: function doubleValue() {},
  ulp: function ulp() {}
};
