// @sitevision/api/types/senselogic/sitevision/api/user/UserIdentityWrapper/index.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports["default"] = void 0;
var _default = exports["default"] = {
  getContacts: function getContacts() {},
  isContact: function isContact() {},
  addContact: function addContact() {},
  removeContact: function removeContact() {},
  getCollaborationGroups: function getCollaborationGroups() {},
  isAdminOf: function isAdminOf() {},
  isMemberOf: function isMemberOf() {},
  leaveGroup: function leaveGroup() {},
  joinGroup: function joinGroup() {},
  followGroup: function followGroup() {},
  getFollowedGroups: function getFollowedGroups() {},
  isFollowing: function isFollowing() {},
  unfollowGroup: function unfollowGroup() {},
  muteGroup: function muteGroup() {},
  unmuteGroup: function unmuteGroup() {},
  isGroupMuted: function isGroupMuted() {},
  muteNotifications: function muteNotifications() {},
  unmuteNotifications: function unmuteNotifications() {},
  isNotificationsMuted: function isNotificationsMuted() {},
  getProfileImage: function getProfileImage() {},
  getProfileImagesFolder: function getProfileImagesFolder() {},
  setProfileImage: function setProfileImage() {},
  setHidden: function setHidden() {},
  setDisabled: function setDisabled() {},
  isDisabled: function isDisabled() {},
  getUserIdentity: function getUserIdentity() {}
};
