// @sitevision/api/types/java/util/Deque/index.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports["default"] = void 0;
var _default = exports["default"] = {
  addFirst: function addFirst() {},
  addLast: function addLast() {},
  offerFirst: function offerFirst() {},
  offerLast: function offerLast() {},
  removeFirst: function removeFirst() {},
  removeLast: function removeLast() {},
  pollFirst: function pollFirst() {},
  pollLast: function pollLast() {},
  getFirst: function getFirst() {},
  getLast: function getLast() {},
  peekFirst: function peekFirst() {},
  peekLast: function peekLast() {},
  removeFirstOccurrence: function removeFirstOccurrence() {},
  removeLastOccurrence: function removeLastOccurrence() {},
  add: function add() {},
  offer: function offer() {},
  remove: function remove() {},
  poll: function poll() {},
  element: function element() {},
  peek: function peek() {},
  push: function push() {},
  pop: function pop() {},
  contains: function contains() {},
  size: function size() {},
  iterator: function iterator() {},
  descendingIterator: function descendingIterator() {}
};
