// @sitevision/api/types/java/math/BigInteger/index.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports["default"] = void 0;
var _default = exports["default"] = {
  nextProbablePrime: function nextProbablePrime() {},
  add: function add() {},
  subtract: function subtract() {},
  multiply: function multiply() {},
  divide: function divide() {},
  divideAndRemainder: function divideAndRemainder() {},
  remainder: function remainder() {},
  pow: function pow() {},
  gcd: function gcd() {},
  abs: function abs() {},
  negate: function negate() {},
  signum: function signum() {},
  mod: function mod() {},
  modPow: function modPow() {},
  modInverse: function modInverse() {},
  shiftLeft: function shiftLeft() {},
  shiftRight: function shiftRight() {},
  and: function and() {},
  or: function or() {},
  xor: function xor() {},
  not: function not() {},
  andNot: function andNot() {},
  testBit: function testBit() {},
  setBit: function setBit() {},
  clearBit: function clearBit() {},
  flipBit: function flipBit() {},
  getLowestSetBit: function getLowestSetBit() {},
  bitLength: function bitLength() {},
  bitCount: function bitCount() {},
  isProbablePrime: function isProbablePrime() {},
  compareTo: function compareTo() {},
  equals: function equals() {},
  min: function min() {},
  max: function max() {},
  hashCode: function hashCode() {},
  toString: function toString() {},
  toByteArray: function toByteArray() {},
  intValue: function intValue() {},
  longValue: function longValue() {},
  floatValue: function floatValue() {},
  doubleValue: function doubleValue() {},
  longValueExact: function longValueExact() {},
  intValueExact: function intValueExact() {},
  shortValueExact: function shortValueExact() {},
  byteValueExact: function byteValueExact() {}
};
