// @sitevision/api/types/java/time/LocalDate/index.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports["default"] = void 0;
var _default = exports["default"] = {
  isSupported: function isSupported() {},
  range: function range() {},
  get: function get() {},
  getLong: function getLong() {},
  getChronology: function getChronology() {},
  getEra: function getEra() {},
  getYear: function getYear() {},
  getMonthValue: function getMonthValue() {},
  getMonth: function getMonth() {},
  getDayOfMonth: function getDayOfMonth() {},
  getDayOfYear: function getDayOfYear() {},
  getDayOfWeek: function getDayOfWeek() {},
  isLeapYear: function isLeapYear() {},
  lengthOfMonth: function lengthOfMonth() {},
  lengthOfYear: function lengthOfYear() {},
  "with": function _with() {},
  withYear: function withYear() {},
  withMonth: function withMonth() {},
  withDayOfMonth: function withDayOfMonth() {},
  withDayOfYear: function withDayOfYear() {},
  plus: function plus() {},
  plusYears: function plusYears() {},
  plusMonths: function plusMonths() {},
  plusWeeks: function plusWeeks() {},
  plusDays: function plusDays() {},
  minus: function minus() {},
  minusYears: function minusYears() {},
  minusMonths: function minusMonths() {},
  minusWeeks: function minusWeeks() {},
  minusDays: function minusDays() {},
  query: function query() {},
  adjustInto: function adjustInto() {},
  until: function until() {},
  format: function format() {},
  atTime: function atTime() {},
  atStartOfDay: function atStartOfDay() {},
  toEpochDay: function toEpochDay() {},
  compareTo: function compareTo() {},
  isAfter: function isAfter() {},
  isBefore: function isBefore() {},
  isEqual: function isEqual() {},
  equals: function equals() {},
  hashCode: function hashCode() {},
  toString: function toString() {}
};
