// @sitevision/api/types/java/time/chrono/ChronoZonedDateTime/index.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports["default"] = void 0;
var _default = exports["default"] = {
  timeLineOrder: function timeLineOrder() {},
  from: function from() {},
  range: function range() {},
  get: function get() {},
  getLong: function getLong() {},
  toLocalDate: function toLocalDate() {},
  toLocalTime: function toLocalTime() {},
  toLocalDateTime: function toLocalDateTime() {},
  getChronology: function getChronology() {},
  getOffset: function getOffset() {},
  getZone: function getZone() {},
  withEarlierOffsetAtOverlap: function withEarlierOffsetAtOverlap() {},
  withLaterOffsetAtOverlap: function withLaterOffsetAtOverlap() {},
  withZoneSameLocal: function withZoneSameLocal() {},
  withZoneSameInstant: function withZoneSameInstant() {},
  isSupported: function isSupported() {},
  "with": function _with() {},
  plus: function plus() {},
  minus: function minus() {},
  query: function query() {},
  format: function format() {},
  toInstant: function toInstant() {},
  toEpochSecond: function toEpochSecond() {},
  compareTo: function compareTo() {},
  isBefore: function isBefore() {},
  isAfter: function isAfter() {},
  isEqual: function isEqual() {},
  equals: function equals() {},
  hashCode: function hashCode() {},
  toString: function toString() {}
};
