// @sitevision/api/types/java/time/Duration/index.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports["default"] = void 0;
var _default = exports["default"] = {
  get: function get() {},
  getUnits: function getUnits() {},
  isZero: function isZero() {},
  isNegative: function isNegative() {},
  getSeconds: function getSeconds() {},
  getNano: function getNano() {},
  withSeconds: function withSeconds() {},
  withNanos: function withNanos() {},
  plus: function plus() {},
  plusDays: function plusDays() {},
  plusHours: function plusHours() {},
  plusMinutes: function plusMinutes() {},
  plusSeconds: function plusSeconds() {},
  plusMillis: function plusMillis() {},
  plusNanos: function plusNanos() {},
  minus: function minus() {},
  minusDays: function minusDays() {},
  minusHours: function minusHours() {},
  minusMinutes: function minusMinutes() {},
  minusSeconds: function minusSeconds() {},
  minusMillis: function minusMillis() {},
  minusNanos: function minusNanos() {},
  multipliedBy: function multipliedBy() {},
  dividedBy: function dividedBy() {},
  negated: function negated() {},
  abs: function abs() {},
  addTo: function addTo() {},
  subtractFrom: function subtractFrom() {},
  toDays: function toDays() {},
  toHours: function toHours() {},
  toMinutes: function toMinutes() {},
  toMillis: function toMillis() {},
  toNanos: function toNanos() {},
  compareTo: function compareTo() {},
  equals: function equals() {},
  hashCode: function hashCode() {},
  toString: function toString() {}
};
