// @sitevision/api/types/java/io/File/index.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports["default"] = void 0;
var _default = exports["default"] = {
  getName: function getName() {},
  getParent: function getParent() {},
  getParentFile: function getParentFile() {},
  getPath: function getPath() {},
  isAbsolute: function isAbsolute() {},
  getAbsolutePath: function getAbsolutePath() {},
  getAbsoluteFile: function getAbsoluteFile() {},
  getCanonicalPath: function getCanonicalPath() {},
  getCanonicalFile: function getCanonicalFile() {},
  toURL: function toURL() {},
  toURI: function toURI() {},
  canRead: function canRead() {},
  canWrite: function canWrite() {},
  exists: function exists() {},
  isDirectory: function isDirectory() {},
  isFile: function isFile() {},
  isHidden: function isHidden() {},
  lastModified: function lastModified() {},
  length: function length() {},
  createNewFile: function createNewFile() {},
  "delete": function _delete() {},
  deleteOnExit: function deleteOnExit() {},
  list: function list() {},
  listFiles: function listFiles() {},
  mkdir: function mkdir() {},
  mkdirs: function mkdirs() {},
  renameTo: function renameTo() {},
  setLastModified: function setLastModified() {},
  setReadOnly: function setReadOnly() {},
  setWritable: function setWritable() {},
  setReadable: function setReadable() {},
  setExecutable: function setExecutable() {},
  canExecute: function canExecute() {},
  getTotalSpace: function getTotalSpace() {},
  getFreeSpace: function getFreeSpace() {},
  getUsableSpace: function getUsableSpace() {},
  compareTo: function compareTo() {},
  equals: function equals() {},
  hashCode: function hashCode() {},
  toString: function toString() {},
  toPath: function toPath() {}
};
