// @sitevision/api/server/SiteCookieUtil/index.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports["default"] = void 0;
var _default = exports["default"] = {
  getNecessarySiteCookies: function getNecessarySiteCookies() {},
  getNecessarySiteCookiesTitle: function getNecessarySiteCookiesTitle() {},
  getNecessarySiteCookiesDescription: function getNecessarySiteCookiesDescription() {},
  getAnalyticsSiteCookies: function getAnalyticsSiteCookies() {},
  getAnalyticsSiteCookiesTitle: function getAnalyticsSiteCookiesTitle() {},
  getAnalyticsSiteCookiesDescription: function getAnalyticsSiteCookiesDescription() {},
  getMarketingSiteCookies: function getMarketingSiteCookies() {},
  getMarketingSiteCookiesTitle: function getMarketingSiteCookiesTitle() {},
  getMarketingSiteCookiesDescription: function getMarketingSiteCookiesDescription() {},
  getFunctionalSiteCookies: function getFunctionalSiteCookies() {},
  getFunctionalSiteCookiesTitle: function getFunctionalSiteCookiesTitle() {},
  getFunctionalSiteCookiesDescription: function getFunctionalSiteCookiesDescription() {},
  getCustomSiteCookies: function getCustomSiteCookies() {},
  getCustomSiteCookiesTitle: function getCustomSiteCookiesTitle() {},
  getCustomSiteCookiesDescription: function getCustomSiteCookiesDescription() {},
  resolveSiteCookieByIdentifier: function resolveSiteCookieByIdentifier() {},
  checkUserConsent: function checkUserConsent() {},
  hasPendingUserConsents: function hasPendingUserConsents() {},
  hasUserConsentCookie: function hasUserConsentCookie() {},
  getUserConsentCookieName: function getUserConsentCookieName() {},
  createUserConsentCookieValue: function createUserConsentCookieValue() {},
  getUserConsents: function getUserConsents() {}
};
