// @sitevision/api/types/java/security/Provider/index.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports["default"] = void 0;
var _default = exports["default"] = {
  getName: function getName() {},
  getVersion: function getVersion() {},
  getInfo: function getInfo() {},
  toString: function toString() {},
  clear: function clear() {},
  load: function load() {},
  putAll: function putAll() {},
  entrySet: function entrySet() {},
  keySet: function keySet() {},
  values: function values() {},
  put: function put() {},
  putIfAbsent: function putIfAbsent() {},
  remove: function remove() {},
  replace: function replace() {},
  replaceAll: function replaceAll() {},
  compute: function compute() {},
  computeIfAbsent: function computeIfAbsent() {},
  computeIfPresent: function computeIfPresent() {},
  merge: function merge() {},
  get: function get() {},
  getOrDefault: function getOrDefault() {},
  forEach: function forEach() {},
  keys: function keys() {},
  elements: function elements() {},
  getProperty: function getProperty() {},
  getService: function getService() {},
  getServices: function getServices() {},
  putService: function putService() {},
  removeService: function removeService() {}
};
