// @sitevision/api/types/java/util/ArrayList/index.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports["default"] = void 0;
var _default = exports["default"] = {
  trimToSize: function trimToSize() {},
  ensureCapacity: function ensureCapacity() {},
  size: function size() {},
  isEmpty: function isEmpty() {},
  contains: function contains() {},
  indexOf: function indexOf() {},
  lastIndexOf: function lastIndexOf() {},
  clone: function clone() {},
  toArray: function toArray() {},
  get: function get() {},
  set: function set() {},
  add: function add() {},
  remove: function remove() {},
  clear: function clear() {},
  addAll: function addAll() {},
  removeRange: function removeRange() {},
  removeAll: function removeAll() {},
  retainAll: function retainAll() {},
  listIterator: function listIterator() {},
  iterator: function iterator() {},
  subList: function subList() {},
  forEach: function forEach() {},
  spliterator: function spliterator() {},
  removeIf: function removeIf() {},
  replaceAll: function replaceAll() {},
  sort: function sort() {}
};
