// @sitevision/api/server/CollectionsInstance/index.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports["default"] = void 0;
var _default = exports["default"] = {
  sort: function sort() {},
  binarySearch: function binarySearch() {},
  reverse: function reverse() {},
  shuffle: function shuffle() {},
  swap: function swap() {},
  fill: function fill() {},
  copy: function copy() {},
  min: function min() {},
  max: function max() {},
  rotate: function rotate() {},
  replaceAll: function replaceAll() {},
  indexOfSubList: function indexOfSubList() {},
  lastIndexOfSubList: function lastIndexOfSubList() {},
  unmodifiableCollection: function unmodifiableCollection() {},
  unmodifiableSet: function unmodifiableSet() {},
  unmodifiableSortedSet: function unmodifiableSortedSet() {},
  unmodifiableList: function unmodifiableList() {},
  unmodifiableMap: function unmodifiableMap() {},
  unmodifiableSortedMap: function unmodifiableSortedMap() {},
  synchronizedCollection: function synchronizedCollection() {},
  synchronizedSet: function synchronizedSet() {},
  synchronizedSortedSet: function synchronizedSortedSet() {},
  synchronizedList: function synchronizedList() {},
  synchronizedMap: function synchronizedMap() {},
  synchronizedSortedMap: function synchronizedSortedMap() {},
  checkedCollection: function checkedCollection() {},
  checkedSet: function checkedSet() {},
  checkedSortedSet: function checkedSortedSet() {},
  checkedList: function checkedList() {},
  checkedMap: function checkedMap() {},
  checkedSortedMap: function checkedSortedMap() {},
  emptySet: function emptySet() {},
  emptyList: function emptyList() {},
  emptyMap: function emptyMap() {},
  singleton: function singleton() {},
  singletonList: function singletonList() {},
  singletonMap: function singletonMap() {},
  nCopies: function nCopies() {},
  reverseOrder: function reverseOrder() {},
  enumeration: function enumeration() {},
  list: function list() {},
  frequency: function frequency() {},
  disjoint: function disjoint() {},
  addAll: function addAll() {},
  newSetFromMap: function newSetFromMap() {},
  asLifoQueue: function asLifoQueue() {},
  emptyIterator: function emptyIterator() {},
  emptyListIterator: function emptyListIterator() {},
  emptyEnumeration: function emptyEnumeration() {},
  unmodifiableNavigableSet: function unmodifiableNavigableSet() {},
  unmodifiableNavigableMap: function unmodifiableNavigableMap() {},
  synchronizedNavigableSet: function synchronizedNavigableSet() {},
  synchronizedNavigableMap: function synchronizedNavigableMap() {},
  checkedQueue: function checkedQueue() {},
  checkedNavigableSet: function checkedNavigableSet() {},
  checkedNavigableMap: function checkedNavigableMap() {},
  emptySortedSet: function emptySortedSet() {},
  emptyNavigableSet: function emptyNavigableSet() {},
  emptySortedMap: function emptySortedMap() {},
  emptyNavigableMap: function emptyNavigableMap() {}
};
