// @sitevision/api/server/ScriptUtil/index.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports["default"] = void 0;
var _default = exports["default"] = {
  getTrue: function getTrue() {},
  getFalse: function getFalse() {},
  getNull: function getNull() {},
  getNonNull: function getNonNull() {},
  getNonBlank: function getNonBlank() {},
  isNotEmpty: function isNotEmpty() {},
  isEmpty: function isEmpty() {},
  isNotBlank: function isNotBlank() {},
  isBlank: function isBlank() {},
  getSystemProperty: function getSystemProperty() {},
  getInstance: function getInstance() {},
  getList: function getList() {},
  getFormatPattern: function getFormatPattern() {},
  getDateAsString: function getDateAsString() {},
  getCalendarAsString: function getCalendarAsString() {},
  getHumanPresentableSize: function getHumanPresentableSize() {},
  stringToInt: function stringToInt() {},
  stringToLong: function stringToLong() {},
  stringToDouble: function stringToDouble() {},
  stringToFloat: function stringToFloat() {},
  isTrue: function isTrue() {},
  isFalse: function isFalse() {},
  joinArray: function joinArray() {},
  joinCollection: function joinCollection() {},
  messageFormat: function messageFormat() {},
  swallow: function swallow() {},
  trimToNull: function trimToNull() {},
  trimToEmpty: function trimToEmpty() {},
  flatten: function flatten() {}
};
