// @sitevision/api/server/LinkRenderer/index.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports["default"] = void 0;
var _default = exports["default"] = {
  update: function update() {},
  updateClean: function updateClean() {},
  clearAllOptional: function clearAllOptional() {},
  render: function render() {},
  setTarget: function setTarget() {},
  setStringTarget: function setStringTarget() {},
  isRenderableTarget: function isRenderableTarget() {},
  isValidTarget: function isValidTarget() {},
  setFontClass: function setFontClass() {},
  setFont: function setFont() {},
  setText: function setText() {},
  setTitle: function setTitle() {},
  removeTitle: function removeTitle() {},
  setStyle: function setStyle() {},
  clearStyle: function clearStyle() {},
  setLang: function setLang() {},
  clearLang: function clearLang() {},
  setHrefLang: function setHrefLang() {},
  clearHrefLang: function clearHrefLang() {},
  addTargetParameter: function addTargetParameter() {},
  clearTargetParameters: function clearTargetParameters() {},
  setUseParameterEncoding: function setUseParameterEncoding() {},
  forceUseParameterEncoding: function forceUseParameterEncoding() {},
  clearUseParameterEncoding: function clearUseParameterEncoding() {},
  addDataAttribute: function addDataAttribute() {},
  clearDataAttributes: function clearDataAttributes() {},
  addAriaAttribute: function addAriaAttribute() {},
  clearAriaAttributes: function clearAriaAttributes() {},
  setAccessKey: function setAccessKey() {},
  clearAccessKey: function clearAccessKey() {},
  setRel: function setRel() {},
  clearRel: function clearRel() {},
  setOnclick: function setOnclick() {},
  clearOnclick: function clearOnclick() {},
  setId: function setId() {},
  clearId: function clearId() {},
  setUseLinkDecorationSettings: function setUseLinkDecorationSettings() {},
  forceUseLinkDecorationSettings: function forceUseLinkDecorationSettings() {},
  clearUseLinkDecorationSettings: function clearUseLinkDecorationSettings() {},
  setUseResourceDecorationSettings: function setUseResourceDecorationSettings() {},
  forceUseResourceDecorationSettings: function forceUseResourceDecorationSettings() {},
  clearUseResourceDecorationSettings: function clearUseResourceDecorationSettings() {},
  setOpenNewWindow: function setOpenNewWindow() {},
  forceOpenNewWindow: function forceOpenNewWindow() {},
  clearOpenNewWindow: function clearOpenNewWindow() {},
  setUseDownload: function setUseDownload() {},
  forceUseDownload: function forceUseDownload() {},
  clearUseDownload: function clearUseDownload() {},
  setUseEncoding: function setUseEncoding() {},
  forceUseEncoding: function forceUseEncoding() {},
  clearUseEncoding: function clearUseEncoding() {},
  setUseAutoTitle: function setUseAutoTitle() {},
  forceUseAutoTitle: function forceUseAutoTitle() {},
  clearUseAutoTitle: function clearUseAutoTitle() {},
  setUseCrossSiteTargetChecking: function setUseCrossSiteTargetChecking() {},
  forceUseCrossSiteTargetChecking: function forceUseCrossSiteTargetChecking() {},
  clearUseCrossSiteTargetChecking: function clearUseCrossSiteTargetChecking() {}
};
