// @sitevision/api/types/java/util/Date/index.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports["default"] = void 0;
var _default = exports["default"] = {
  clone: function clone() {},
  getYear: function getYear() {},
  setYear: function setYear() {},
  getMonth: function getMonth() {},
  setMonth: function setMonth() {},
  getDate: function getDate() {},
  setDate: function setDate() {},
  getDay: function getDay() {},
  getHours: function getHours() {},
  setHours: function setHours() {},
  getMinutes: function getMinutes() {},
  setMinutes: function setMinutes() {},
  getSeconds: function getSeconds() {},
  setSeconds: function setSeconds() {},
  getTime: function getTime() {},
  setTime: function setTime() {},
  before: function before() {},
  after: function after() {},
  equals: function equals() {},
  compareTo: function compareTo() {},
  hashCode: function hashCode() {},
  toString: function toString() {},
  toLocaleString: function toLocaleString() {},
  toGMTString: function toGMTString() {},
  getTimezoneOffset: function getTimezoneOffset() {},
  toInstant: function toInstant() {}
};
