// @sitevision/api/server/NodeTypeUtil/index.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports["default"] = void 0;
var _default = exports["default"] = {
  isLayout: function isLayout() {},
  isLink: function isLink() {},
  isPage: function isPage() {},
  isPortlet: function isPortlet() {},
  isArchive: function isArchive() {},
  isArticle: function isArticle() {},
  isFolder: function isFolder() {},
  isFile: function isFile() {},
  isImage: function isImage() {},
  isTemporaryFile: function isTemporaryFile() {},
  isUser: function isUser() {},
  isSimpleUser: function isSimpleUser() {},
  isSystemUser: function isSystemUser() {},
  isUserIdentity: function isUserIdentity() {},
  isSite: function isSite() {},
  isSitePage: function isSitePage() {},
  isTemplate: function isTemplate() {},
  isTrashcan: function isTrashcan() {},
  isColor: function isColor() {},
  isDecoration: function isDecoration() {},
  isDirectory: function isDirectory() {},
  isFont: function isFont() {},
  isStructureFolder: function isStructureFolder() {},
  isStructureLink: function isStructureLink() {},
  isStructurePage: function isStructurePage() {},
  isCollaborationGroup: function isCollaborationGroup() {},
  isCollaborationGroupFolder: function isCollaborationGroupFolder() {},
  isCollaborationGroupPage: function isCollaborationGroupPage() {},
  isCollaborationGroupTemplate: function isCollaborationGroupTemplate() {},
  isApplicationIndex: function isApplicationIndex() {},
  isNodeIndex: function isNodeIndex() {},
  isRssFeed: function isRssFeed() {},
  isRssFeedItem: function isRssFeedItem() {},
  isRssFeedRepository: function isRssFeedRepository() {},
  isAnyRenderableType: function isAnyRenderableType() {},
  isAnyContentType: function isAnyContentType() {},
  isAnyUserType: function isAnyUserType() {},
  isAnyMetadataDefinitionType: function isAnyMetadataDefinitionType() {},
  isType: function isType() {},
  isTypeOf: function isTypeOf() {}
};
