// @sitevision/api/types/java/time/chrono/Chronology/index.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports["default"] = void 0;
var _default = exports["default"] = {
  from: function from() {},
  ofLocale: function ofLocale() {},
  of: function of() {},
  getAvailableChronologies: function getAvailableChronologies() {},
  getId: function getId() {},
  getCalendarType: function getCalendarType() {},
  date: function date() {},
  dateYearDay: function dateYearDay() {},
  dateEpochDay: function dateEpochDay() {},
  dateNow: function dateNow() {},
  localDateTime: function localDateTime() {},
  zonedDateTime: function zonedDateTime() {},
  isLeapYear: function isLeapYear() {},
  prolepticYear: function prolepticYear() {},
  eraOf: function eraOf() {},
  eras: function eras() {},
  range: function range() {},
  getDisplayName: function getDisplayName() {},
  resolveDate: function resolveDate() {},
  period: function period() {},
  compareTo: function compareTo() {},
  equals: function equals() {},
  hashCode: function hashCode() {},
  toString: function toString() {}
};
