// @sitevision/api/types/java/nio/file/spi/FileSystemProvider/index.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports["default"] = void 0;
var _default = exports["default"] = {
  getScheme: function getScheme() {},
  newFileSystem: function newFileSystem() {},
  getFileSystem: function getFileSystem() {},
  getPath: function getPath() {},
  newInputStream: function newInputStream() {},
  newOutputStream: function newOutputStream() {},
  newFileChannel: function newFileChannel() {},
  newAsynchronousFileChannel: function newAsynchronousFileChannel() {},
  newByteChannel: function newByteChannel() {},
  newDirectoryStream: function newDirectoryStream() {},
  createDirectory: function createDirectory() {},
  createSymbolicLink: function createSymbolicLink() {},
  createLink: function createLink() {},
  "delete": function _delete() {},
  deleteIfExists: function deleteIfExists() {},
  readSymbolicLink: function readSymbolicLink() {},
  copy: function copy() {},
  move: function move() {},
  isSameFile: function isSameFile() {},
  isHidden: function isHidden() {},
  getFileStore: function getFileStore() {},
  checkAccess: function checkAccess() {},
  getFileAttributeView: function getFileAttributeView() {},
  readAttributes: function readAttributes() {},
  setAttribute: function setAttribute() {}
};
